### PR TITLE
Line alignments and some text corrections

### PR DESCRIPTION
**Pull request by TheCartpenter**

### CODE REVIEW

**Comment by TheCartpenter** [comment]
> It seems to me that the user wants to cheat statistics. For the sake of a small change that does not affect the code (such as adding / removing a space), he makes a separate commit for each of the changed files. Why not combine all the little things into one commit?

And now that I did solved the issue with the repository, you're still complaining about the fact that commits 'should' be merged while they have indeed been merged since today. Unless you are looking for me to post these commits individually, then there's no reason for this post to be addressed about, since these commits have been merged since the issue was resolved today.

**Comment by batumibiz** [comment]
Okay, let's take a look at the details. Look at:

- this: https://github.com/opencart/opencart/pull/12620/commits/94cd1306d929f3c6a6e124caef673cbb15a971f7
- and this: https://github.com/opencart/opencart/pull/12620/commits/6f86c14846212f7c53592e26c9c8ddf4252420e8
- and this: https://github.com/opencart/opencart/pull/12620/commits/6f86c14846212f7c53592e26c9c8ddf4252420e8
- and this: https://github.com/opencart/opencart/pull/12620/commits/9fbcb57e6f4db735fb0a49f64c8f3dc049a603e0
- and this: https://github.com/opencart/opencart/pull/12620/commits/5bfb80b13ac7971ef17835e6caad04de6a9abc74
...
- and this: https://github.com/opencart/opencart/pull/12620/commits/db2eef9af1f4234747112b9caa2e08d8e7a5ed1b

Why do you create a separate commit for each file for the sake of adding / removing one space that does not affect code execution in any way?

**Comment by TheCartpenter** [comment]
The issue has now been closed: https://github.com/opencart/opencart/issues/12616#issuecomment-1675024743 . Old news.

**Comment by batumibiz** [comment]
> The issue has now been closed: [#12616 (comment)](https://github.com/opencart/opencart/issues/12616#issuecomment-1675024743) . Old news.

And what about Issue?
I'm talking about your PR

**Comment by TheCartpenter** [comment]
> > The issue has now been closed: [#12616 (comment)](https://github.com/opencart/opencart/issues/12616#issuecomment-1675024743) . Old news.
> 
> And what about Issue? I'm talking about your PR

You're sticking back to the past based on questions I already answered previously. Get over it.

**Comment by mhcwebdesign** [comment]
To be fair, these changes don't hurt, and make the source files slightly more readable.

So let's not argue about it, and focus on the more important bugfixes and issues, of which there are still plenty in OpenCart 4.

**Comment by TheCartpenter** [comment]
> To be fair, these changes don't hurt, and make the source files slightly more readable.
> 
> So let's not argue about it, and focus on the more important bugfixes and issues, of which there are still plenty in OpenCart 4.

Couldn't agree more. The most important, however, is about keeping the same code structures since this, after all, about a CMS platform.

**Comment by batumibiz** [comment]
> To be fair, these changes don't hurt, and make the source files slightly more readable.
> 
> So let's not argue about it, and focus on the more important bugfixes and issues, of which there are still plenty in OpenCart 4.

@mhcwebdesign I absolutely agree that it is useful to make code readable.
But I (and many other developers) complain about the bad behavior of TheCartpenter, which spams with its useless commits and makes it difficult to focus on really important things. Look at the commits of this PR, look at the list of closed PRs and you will understand me.

**Comment by TheCartpenter** [comment]
> > To be fair, these changes don't hurt, and make the source files slightly more readable.
> > So let's not argue about it, and focus on the more important bugfixes and issues, of which there are still plenty in OpenCart 4.
> 
> @mhcwebdesign I absolutely agree that it is useful to make code readable. But I (and many other developers) complain about the bad behavior of TheCartpenter, which spams with its useless commits and makes it difficult to focus on really important things. Look at the commits of this PR, look at the list of closed PRs and you will understand me.

You're the one who's poisoning my commits right now. It was combined and you just keep bringing on the subject after I explained the situation. Since you refuse to understand the situation, from now on, please step away from my commits in the future!

**Comment by batumibiz** [comment]
@TheCartpenter if you stop spamming and start committing for a really important reason, I have your full support.

**Comment by TheCartpenter** [comment]
> @TheCartpenter if you stop spamming and start committing for a really important reason, I have your full support.

The commits are back. That's on you to keep complaining about it.

**Comment by TheCartpenter** [comment]
> @TheCartpenter if you stop spamming and start committing for a really important reason, I have your full support.

It was already done. Yet, you chose to remain this way.

File: upload/admin/language/en-gb/setting/setting.php [hunk 1]
```diff
@@ -39,7 +39,7 @@
 $_['text_captcha']                        = 'Captcha';
 $_['text_hour']                           = 'Hour(s)';
 $_['text_none']                           = 'None';
-$_['text_lax']                            = 'Lax' ;
```
Comment (TheCartpenter) [inline upload/admin/language/en-gb/setting/setting.php:42]: https://github.com/opencart/opencart/commit/f533509a578d176d3e961d1fd207b77b169758f9#diff-2a027bf142934be77551b0ec117750441495667e867bd2c02b641e26d47f1277L44

**Pull request closed without merging**